[WIP]Create CONTRIBUTING.md (#20)
sensu/sensu-go-plugin: CONTRIBUTING.md

# Contributing

> Contributions to this plugin are welcome. To contribute a feature or change to this plugin, please follow the steps below

Reviewer: ```suggestion
Contributions to this plugin are welcome. To contribute a feature or change, please follow these steps:
```

> 1. Open an issue to describe your proposed improvement or feature, or describe an issue with the plugin

Reviewer: ```suggestion
1. Open an issue to describe a proposed improvement/feature or a problem with the plugin.
```

> 2. Fork this respository

Reviewer: ```suggestion
2. Fork this respository.
```

> 3. Create your feature or fix branch (`git checkout -b my-new-feature`)

Reviewer: ```suggestion
3. Create your feature or fix branch (`git checkout -b my-new-feature`).
```

> 4. If applicable, add a [CHANGELOG.md entry](#changelog) describing your change.

Reviewer: ```suggestion
4. Add a [CHANGELOG.md entry](#changelog) to describe your change.
```

Reviewer: If we say "If applicable" I think we need to tell them how to figure out if it's applicable.

> 5. Push your feature or fix branch (`git push origin my-new-feature`)

Reviewer: ```suggestion
5. Push your feature or fix branch (`git push origin my-new-feature`).
```

> 6. Create a Pull Request as appropriate based on the issue discussion

Reviewer: ```suggestion
6. Create a pull request for your feature or fix. Include the link to your issue in the pull request description.
```
